src/components/FourOhFour/FourOhFour.jsx

```diff
@@ -43,7 +43,12 @@ const FourOhFour = ({ strings, msg }) => (
   <Styled>
     <div className="container">
       <Helmet title={`${strings.error} 404`} />
-      <img src="/assets/images/sad.gif" alt="" />
+      <img
+        src={`/assets/images/${
+          sample(['sad', 'bawl', 'cry', 'charm-cry'])
```

Comment:
it might be possible to just use `Math.random()` instead

Reply:
Thank you @Argn0 , I have switch from lodash method to Math.random() thingy 💯